src/templates/kit/screens/VotingScreen.js

```diff
@@ -126,7 +126,11 @@ function VotingScreen({
           quorum: Math.floor(quorum),
           duration,
         }
-        next(dataKey ? { ...data, [dataKey]: screenData } : screenData)
+        next(
+          dataKey
+            ? { ...data, [dataKey]: screenData }
+            : { ...data, ...screenData }
```

Comment:
If you want a one liner: `{ ...data, ...(dataKey? {[dataKey]: screenData} : screenData) }`